README.md

```diff
@@ -53,6 +53,7 @@ Refer [hello_world.ipynb](https://github.com/mozilla/overscripted/blob/master/an
 
 - TLD means Top-level Domain. You can read up more about it [here.](https://en.wikipedia.org/wiki/Top-level_domain) 
 - [Web Crawler](https://en.wikipedia.org/wiki/Web_crawler)- It is a program or automated script which browses the World Wide   Web in a methodical, automated manner.
+- [Fingerprinting](https://en.wikipedia.org/wiki/Device_fingerprint) is the process of creating a unique identifier based off   of some characteristics of your hardware, operating system and browser. The basic idea behind it is that JavaScript can       access a bunch of different APIs that your browser has implemented. When used correctly, these allow webpages to be     rendered properly
```

Comment:
* Please remove excess whitespace.
* The second sentence does not conclude.
* Please avoid judgements like "correctly" unless you can define correctly and justify why a given use is "incorrect." (I would suggest avoiding judgement.)

Reply:
Ready for Review @birdsarah. 
Apologies for the excess white spaces and I will keep in mind to avoid judgments.
Thanks